uber/config.py

```diff
@@ -106,18 +106,21 @@ def get_attendee_price(self, dt=None):
         price = self.INITIAL_ATTENDEE
         if self.PRICE_BUMPS_ENABLED:
 
+            for day, bumped_price in sorted(self.PRICE_BUMPS.items()):
+                if (dt or sa.localized_now()) >= day:
```

Comment:
one other minor note: previously we were using ```datetime.now(UTC)``` here, now we're using ```sa.localized_now()```.  Seems like it should be fine because if we mixed up the timezones, we're only going to be off by 5 hours and the specific price bump hours are not really that important.